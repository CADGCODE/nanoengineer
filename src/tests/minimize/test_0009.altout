======= test_0009.test =======
TYPE struct
======= exitvalue =======
0
======= structurematch =======
0
======= stderr =======
== structure comparison ==
======= stdout =======
iters per frame = 10
number of frames = 100
timestep = 1.000000e-16
temp = 0.000000
dump as text
< test_0009.mmp  > test_0009.xyz

Total Ke = 1.117800e-19
final RMS gradient=0.944578 after 66 iterations
== structure comparison ==
at: translate (0.000000 0.000000 0.000000)
       rotate (0.000000 0.000000 0.000000)
        scale 1.000000,
standard deviation = 0.000000e+00, max delta = 0.000000e+00
======= test_0009.trc =======
# nanoENGINEER-1.com Simulator Trace File, Version 050310
#
# Input File:test_0009.mmp
# Output File: test_0009.xyz
# Trace File: test_0009.trc
# Number of Atoms: 18
# Number of Frames: 100
# Steps per Frame: 10
# Temperature: 300.0
# 
# 0 columns:
#

# rms force, high force
1 604.90 771.11
2 474.73 609.79
3 381.15 416.14
4 317.11 397.33
5 269.00 298.43
6 243.68 279.08
7 210.99 247.42
8 192.83 225.92
9 164.54 210.96
10 151.00 196.09
11 137.21 183.66
12 122.83 167.24
13 108.58 154.15
14 97.97 134.45
15 89.98 132.65
16 85.29 117.75
17 75.84 116.67
18 71.40 109.18
19 67.19 105.32
20 63.12 97.46
21 60.00 94.60
22 57.03 87.64
23 54.85 86.76
24 53.36 79.69
25 51.00 80.03
26 49.63 76.12
# Switching to Conjugate-Gradient
27 48.92 0.00
28 41.46 0.00
29 39.79 0.00
30 39.74 0.00
31 39.50 0.00
32 30.56 0.00
33 26.09 0.00
34 26.02 0.00
35 24.59 0.00
36 20.48 0.00
37 20.33 0.00
38 17.41 0.00
39 15.60 0.00
40 14.75 0.00
41 12.69 0.00
42 10.58 0.00
43 7.85 0.00
44 7.65 0.00
45 7.10 0.00
46 6.45 0.00
47 5.92 0.00
48 5.64 0.00
49 5.33 0.00
50 5.04 0.00
51 4.81 0.00
52 4.34 0.00
53 3.71 0.00
54 3.45 0.00
55 2.76 0.00
56 2.55 0.00
57 2.18 0.00
58 2.10 0.00
59 1.91 0.00
60 1.79 0.00
61 1.63 0.00
62 1.39 0.00
63 1.35 0.00
64 1.12 0.00
65 0.94 0.00
66 0.94 nan
# Done: Minimization final rms: 0.944578, highForce: nan
======= test_0009.xyz =======
18
RMS=0.944578
C -2.919195 0.308230 -0.009946
H -3.543230 0.932399 -0.644376
H -2.296592 -0.314696 -0.646957
C -2.021851 1.202030 0.882969
H -2.644673 1.840587 1.503990
H -1.397204 1.834823 0.257833
C -3.813202 -0.588662 0.883246
H -4.446065 -1.213329 0.258202
H -4.451814 0.034209 1.504199
C -2.921666 -1.482009 1.782727
H -3.548254 -2.111888 2.408880
H -2.302256 -2.123791 1.161579
C -1.128604 0.310414 1.782472
H -0.486726 -0.309034 1.161423
H -0.498694 0.937042 2.408554
C -2.019861 -0.590962 2.674507
H -2.639215 0.028700 3.317716
H -1.392897 -1.218064 3.302981
