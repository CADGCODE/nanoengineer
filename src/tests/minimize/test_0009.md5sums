stdout AF5CBD846705F1443DA5C6DC75DB4DD6
stderr D41D8CD98F00B204E9800998ECF8427E
test_0009.xyz 39D4A07FF85ADFF5348D004F3E4B06E0
