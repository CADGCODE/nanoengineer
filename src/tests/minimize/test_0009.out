======= test_0009.test =======
======= exitvalue =======
0
======= stderr =======
Warning: partial minimization in CG
======= stdout =======
iters per frame = 10
number of frames = 100
timestep = 1.000000e-16
temp = 0.000000
dump as text
< test_0009.mmp  > test_0009.xyz

Total Ke = 1.117800e-19
final RMS gradient=43.834339 after 14 iterations
======= test_0009.trc =======
# nanoENGINEER-1.com Simulator Trace File, Version 050310
#
# Input File:test_0009.mmp
# Output File: test_0009.xyz
# Trace File: test_0009.trc
# Number of Atoms: 18
# Number of Frames: 100
# Steps per Frame: 10
# Temperature: 300.0
# 
# 0 columns:
#

# rms force, high force
1 1307.96 1365.85
2 783.10 946.67
3 490.00 637.72
4 346.56 434.12
5 211.57 263.82
6 100.59 114.64
7 80.98 102.87
8 305.19 529.85
9 245.24 414.59
10 44.02 67.44
# Switching to Conjugate-Gradient
11 44.02 67.44
12 43.83 67.28
13 99429.15 197152.47
14 43.83 3272480354115239376405119387045373251105587200.00
# Warning: partial minimization in CG
# Done: 
======= test_0009.xyz =======
18
RMS=43.834339
C -2.894570 0.283666 0.012926
H -3.521126 0.910241 -0.659239
H -2.229501 -0.381633 -0.582668
C -2.046392 1.178028 0.905765
H -2.712613 1.776163 1.567451
H -1.419313 1.848083 0.277565
C -3.789061 -0.564178 0.906056
H -4.459537 -1.191026 0.278113
H -4.387235 0.102032 1.567753
C -2.900640 -1.458095 1.759958
H -3.532830 -2.120678 2.391097
H -2.243274 -2.065758 1.098064
C -1.152675 0.289500 1.759760
H -0.544902 -0.367845 1.097929
H -0.489782 0.921463 2.390776
C -2.040846 -0.570055 2.649121
H -2.699232 0.088402 3.259259
H -1.408471 -1.202310 3.310313
